src/components/compound/DlCharts/charts/DlConfusionMatrix/DlConfusionMatrix.vue

```diff
@@ -575,9 +583,8 @@ export default defineComponent({
     grid-template-columns: repeat(var(--matrix-rows), 1fr);
 
     &__cell {
-        font-size: max(calc(var(--cell-dimensions) * 0.1), 12px);
+        font-size: calc(var(--cell-dimensions) * 0.33);
```

Comment:
@prodan-eric have you tested the case of one and two matricies in the demos ? 
does this 0.33 work well ?

Reply:
Yeah, it's the best font scaling I could come up with. Seems to work well in all cases